service/gcs/main.go

```diff
@@ -212,8 +221,8 @@ func main() {
 	oomFile := os.NewFile(oom, "cefd")
 	defer oomFile.Close()
 
-	go readMemoryEvents(gefdFile, "/gcs", int64(*gcsMemLimitBytes), gcsControl)
-	go readMemoryEvents(oomFile, "/containers", containersLimit, containersControl)
+	go readMemoryEvents(gefdFile, "gcs", int64(*gcsMemLimitBytes), gcsControl)
```

Comment:
Why are we changing the cgroup names here?

Reply:
This is just what gets printed out during logs. If it'd be more helpful to see "/gcs" vs. "gcs" then I can change back. It just looked better in the fmt.Sprintf call also haha

Reply:
I think we should keep `/gcs` as that matches the cgroup names used above.

Reply:
That's just the path we pass in to StaticPath, it could also be just "gcs" or "containers". I'll revert this though, I think the path is nicer after looking again